app/src/main/java/com/appunite/loudius/ui/login/LoginScreen.kt

```diff
@@ -149,8 +183,9 @@ fun LoginScreenPreview() {
     }
 }
 
-@Preview(showSystemUi = true, showBackground = true)
 @Composable
+@Preview(showSystemUi = true, showBackground = true, group = "Dialogs")
+@ShowkaseComposable(skip = true)
```

Comment:
**Info:** Showkase has an [issue](https://github.com/airbnb/Showkase/issues/235) showing dialogs in the component browser.